README.md

```diff
@@ -0,0 +1,21 @@
+# foo-config-loader
+
+A config loader for the Foo project.
+
+
+# Project tasks
+
+## In order to setup a local env run...:
```

Comment:
cosmetics: are those `...` needed here?

Reply:
I understand the necessity of the commands below, but how about going one step further and preparing a single bash-script. e.g.called  `setup_dev_env.sh` that would call all necessary commands to setup this project? 
It could be one file that could be used in different contexts like: preparing development environment for developers or preparing tests environment on any machines/containers that could be used in tests. 
What Do You Think (WDYT?)